modules/visitedMenu.js

```diff
@@ -0,0 +1,18 @@
+const links = document.querySelectorAll('.links li');
+
+function resetLinks() {
+  links.forEach((li) => {
+    li.classList.remove('active');
+  });
+}
+
+function initVisitedMenu() {
+  links.forEach((li) => {
+    li.addEventListener('click', () => {
+      resetLinks();
+      li.classList.add('active');
+    });
+  });
+}
```

Comment:
- [ ] Please change these functions to arrow functions as requested in the project requirements.

![image](https://github.com/erickma1/awesome_books_es6/assets/83803414/29c32d1d-dea3-4970-aa8b-8c88d3b52b76)